pkgs/development/libraries/ncurses/default.nix

```diff
@@ -13,7 +13,7 @@
 stdenv.mkDerivation rec {
   # Note the revision needs to be adjusted.
   version = "6.3";
-  name = "ncurses-${version}" + lib.optionalString (abiVersion == "5") "-abi5-compat";
+  pname = "ncurses" + lib.optionalString (abiVersion == "5") "-abi5-compat";
```

Comment:
I am not sure if this should be moved to version.

Reply:
maybe @Artturin, @AndersonTorres or @trofi could suggest on how to proceed.

Reply:
From the POV of Nix interpreter, this is merely cosmetical.
I particularly prefer pname, if there is any reason to put different names.

Reply:
+1 for `pname`. That way at least for me it would be slightly less scary to use `lib.versionAtLeast` and friends against it :)